_posts/2018-03-26-proxstar.md

```diff
@@ -0,0 +1,27 @@
+---
+layout: post
+title: Proxstar
+date: 2018-03-26
+categories: projects
+description: Proxstar is a VM self-service tool that integrates with Proxmox and STARRS
+author: Jordan Rodgers
+author-image: https://avatars1.githubusercontent.com/u/5183608?s=460&v=4
+author-bio: Fourth Year Networking & Systems Administration Student at Rochester Institute of Technology
+author-email: com6056@csh.rit.edu
+author-social:
+  github: https://github.com/com6056
+---
+
+## The Problem
+CSH currently uses Proxmox for VM hosting. It has been working great for everything we use it work with one exception: member VM self-service. The interface isn't very intuitive to someone who might not have created a VM before, creating a higher barrier to entry than we would like for members who might want to start learning how to use Linux for the first time. It also lacked the ability to enforce resource limits and automatically cleanup VMs that members weren't using anymore. All of my previous major projects were much more ops and/or security focused so I figured writing something to solve this problem would be a great way to do a much more development focused one and expand my developer skillset.
```

Comment:
"for everything we use it (with|for)", I think?

Reply:
I would also probably write out "operations" instead of "ops" here.

Reply:
Also, `s/one/project/`

Reply:
You should probably also explain what STARRS is here. Our members may know what it is, but anyone outside CSH won't.